tests/test_mempool.py

```
@@ -0,0 +1,313 @@
+        fee=4,
+    )
+
+    mempool.add_transaction(tx_a1, state)
+    mempool.add_transaction(tx_b0, state)
+    mempool.add_transaction(tx_a0, state)
+    mempool.add_transaction(tx_c0, state)
+
+    selected = mempool.get_transactions_for_mining(state, limit=4)
+
+    assert [tx.fee for tx in selected] == [8, 4, 1, 10]
+    assert selected[2].sender == sender_a and selected[2].nonce == 0
```

Ruff (0.15.1)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)